frontend/app/src/store/defi/const.ts

```diff
@@ -19,6 +19,7 @@ const ARAGON = 'Aragon';
 export const AAVE = 'Aave';
 const BALANCER = 'balancer';
 export const COMPOUND = 'Compound';
+const COMPOUND_COMP = 'Compound Governance';
```

Comment:
@kelsos this is the only thing to do on the frontend, right?

Reply:
looks good :+1: